aptible/resource_app.go

```diff
@@ -260,15 +260,15 @@ func scaleServices(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*aptible.Client)
 	appID := int64(d.Get("app_id").(int))
 
-	services := d.Get("service").(*schema.Set).List()
-
+	if !d.HasChange("service") {
```

Comment:
i'd leave a comment in this block as to why this is being done

Reply:
i think you might want to test this with 3-4 services on one app, and make sure that multiple services (in a list) do not result in a HasChange that's falsely positive if the backend reorders this for any reason (might be wrong or maybe i misunderstand some bits about our backend)

Reply:
I'm not testing or trying to fix the potential issue of reordering right now. Just the glaring issue of scaling all services when none of them have changed.